app/views/shared/_cohort.haml

```diff
@@ -0,0 +1,6 @@
+%li= link_to "All", root_path
+- cohorts.each do |cohort|
+  - if cohort.upcase == ENV['CLASS_ONE']
```

Comment:
Can we just make this alternate between `class-one` and `class-two` CSS classes, e.g. by checking if an index number is even or odd?

That seems like it would be more future-proof when we have different class names in later cohorts.